### ISSUE TITLE
TheSkyX Support

### ISSUE THREAD

**Issue by elendil-software** [mworion/MountWizzard3-DIST#16]
Hi Michel

Do you plan to support TheSkyX in MountWizzard3 ?

If needed I can help for the test/devlopment as I have already done for the version 2.x

Regards

Julien

**Comment by mworion** [comment]
Hi Julien,

so far I did not have this on my shortlist. There might be one topic to be clarified (if you could check this) on TSX side: MW3 needs image integration/downloading and platesolving at the same time. Or better to say: these processes run in parallel in MW3 (that's the main improvement). If TSX can't support this, it will be hard to enable MW3 to work with TSX

Michel

**Comment by elendil-software** [comment]
Hi,

I don't understand, how can you take the image, download it, and platesolve it at the same time ?

Julien

**Comment by mworion** [comment]
Julien, roughly spoken:
There are three queues running in parallel:
Image integration (means no slew)
slewing and downloading (means no integration)
plate solving (could be done any time)
You only have to be clear, that all parts match together.
In result ideally modeling takes only the pure image integration time, because you could do all the other task in parallel.

Michel

**Comment by elendil-software** [comment]
Ok, I will try to see if it's possible

**Comment by elendil-software** [comment]
I've done some tests. That doesn't look good.
The answer is : yes in some cases.

The exposure can be run asynchronously but not the plate solve.
That means that you can start the exposure and just after start a plate solve --> No problem
But if you start by a plate solve, you won't be able to start the exposure before the end of the solve operation. TheSkyX will answer that another script is running.

Regards

**Comment by mworion** [comment]
Hi Julien,

than we stick to the sequential modeling work which is implemented in MW2. Hope this does not matter.

BTW: I moved recently to EKOS with KStars. It's doing most of TSX, it's free and fully client server.

Michel

Closing point without action
